src/patterns/panel/Panel.tsx

```diff
@@ -28,7 +28,8 @@ const Panel: Panel = ({
       <Card.Content>
         {label ? (
           <Card.Heading
-            className={classNames('nhsuk-u-font-size-24', labelProps?.className)}
+            className={classNames('nhsuk-heading-m', labelProps?.className)}
+            headingLevel="h2"
```

Comment:
explicitly setting this to a h2 seems like a bad idea

Reply:
Might be worth putting into props and defaulting

Reply:
Done, and added tests for it

Reply:
Didn't need to hook it up explicitly as the `labelProps.headingLevel` will override the default `headingLevel="h2"` if specified.
But noticed an issue with the className in that `nhsuk-heading-m` was never getting through to being rendered, so fixed that and added a test

Reply:
I've changed this now so that it doesn't provide a default, now passes straight through to the `Card.Heading` which does the default to `h2` if not specified. Also removed the `className` from being hardcoded to `nhsuk-heading-m` as it should be up to the consumer to pass this in, can vary on content v transactional layout

Reply:
works for me